pytest_rts/tests/helper_project/.gitignore

```diff
@@ -1,3 +1,6 @@
 .coverage
-rts-coverage.1234
+rts-coverage.db
 *__pycache__*
+runpytest-*
+stderr
+stdout
```

Comment:
What are stderr and stdout here?

Reply:
Files produced by the `pytester` plugin. I thought ignoring them would make debugging test cases for running working directory changes easier.